src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, num) {
+      callback.call(this, num);
+
+      return this;
+    },
+
+    add(num) {
+      this.result += num;
+
+      return this.result;
```

Comment:
change this.result in operate and reset methods only  
```suggestion
      return this.result + num;
```